01 Modules/00 Built in/README.md

```diff
@@ -55,7 +60,75 @@ Summary steps:
 
 ```
 
--
+- Node.js has a set of `built-in` modules without install any third library. Node has an API for each version, for example [v6.x](https://nodejs.org/docs/latest-v6.x/api/documentation.html), [v8.x](https://nodejs.org/dist/latest-v8.x/docs/api/documentation.html), etc.
+
+- As example we are going to require [`os`](https://nodejs.org/docs/latest-v6.x/api/os.html) module. This module provides some `operating system-related` utility methods:
+
+### ./index.js
+
+```diff
++ const os = require('os');
+
+```
+
+- The `hostname` _method returns the hostname of the operating system as a string._:
+
+### ./index.js
+
+```diff
+const os = require('os');
+
++ console.log(`Host: ${os.hostname()}`);
+
+```
+
+- Press `F5` key to run app:
+
+![hostname method](../../99%20Resources/01%20Modules/00%20Built%20in/hostname%20method.png)
+
+- The `loadavg` _method returns an array containing the 1, 5, and 15 minute load averages._ In this case, is a `UNIX-specific` concept, so is not compatible with Windows platforms.
+
+### ./index.js
+
+```diff
+const os = require('os');
+
+console.log(`Host: ${os.hostname()}`);
++ console.log(`15 min. load avarage ${os.loadavg()}`);
+
+```
+
+- `loadavg` in Windows, press `F5` key to run app:
+
+![loadavg in windows](../../99%20Resources/01%20Modules/00%20Built%20in/loadavg%20in%20windows.png)
+
+- `loadavg` in UNIX, press `F5` key to run app:
+
+![loadavg in unix](../../99%20Resources/01%20Modules/00%20Built%20in/loadavg%20in%20unix.png)
```

Comment:
Pending to add unix screenshot. Same results using `bash` or `cmd` terminal on Windows